src/app/app.component.ts

```diff
@@ -8,4 +9,7 @@ import { Component } from '@angular/core';
 export class AppComponent {
   title = 'content-sharing-app';
 
+  constructor() {
+    console.log(environment.firebaseConfig);
```

Comment:
Is the console log required for the deployment code?

Reply:
Yes definitely :)

Reply:
Will fix it now